assets/css/components/_checkbox-switch.scss

```diff
@@ -0,0 +1,78 @@
+.checkbox-switch {
+  display: inline-block;
+  overflow: hidden;
+  padding: 0.5em 0.5em 0.5em 70px;
+  position: relative;
+}
+
+.checkbox-switch span::before,
+.checkbox-switch span::after {
+  border: 1px solid #747474;
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+.checkbox-switch span::after {
+  background: #eee;
+  border-radius: 100%;
+  content: "";
+  height: 1.5em;
+  left: 0.25em;
+  transition: right 0.1825s ease-in-out;
+  width: 1.5em;
+}
+
+.checkbox-switch span::before {
+  background: rgba(121, 0, 0, 0.3);
+  border-radius: 1.75em;
+  color: #fff;
+  font-variant: small-caps;
+  font-weight: bold;
+  height: 1.5em;
+  left: 0.25em;
+  line-height: normal;
+  transition: background 0.2s ease-in-out;
+  width: 3.75em;
+}
+
+.checkbox-switch input:not(:checked) + span::before {
+  color: #000;
+  content: "off";
+  padding-left: 26px;
```

Comment:
What does the pixel unit represent here? Is it related to the `76px` value on line 4?